android/modules/ui/src/java/ti/modules/titanium/ui/widget/tabgroup/TiUIBottomNavigationTabGroup.java

```diff
@@ -79,13 +80,18 @@ public void addTab(TabProxy tabProxy)
 		}
 	}
 
+	@SuppressLint("PrivateResource")
 	@Override
 	public void addViews(TiBaseActivity activity)
 	{
 		// Manually calculate the proper position of the BottomNavigationView.
-		int resourceID = activity.getResources().getIdentifier("design_bottom_navigation_height", "dimen",
-															   activity.getPackageName());
-		this.mBottomNavigationHeightValue = activity.getResources().getDimensionPixelSize(resourceID);
+		this.mBottomNavigationHeightValue
+			= activity.getResources().getDimensionPixelSize(R.dimen.design_bottom_navigation_height);
+
+		if (activity.getTheme().toString().indexOf("Material3") > -1) {
```

Comment:
This looks hacky. Do we even need a manually-set height? I would prefer to let the system decide the value and use constraints to automatically align other views that are dependent on the height.

Reply:
that would be the best of course. Currently the SDK is using

https://github.com/tidev/titanium-sdk/blob/dd18cd1479a8240ab74474c727e87c48c2b584d0/android/modules/ui/src/java/ti/modules/titanium/ui/widget/tabgroup/TiUIBottomNavigationTabGroup.java#L88-L89

for the BottomNav height (https://github.com/tidev/titanium-sdk/pull/10358) which is causing the issue in Material 3 as they've renamed the variable with the proper height.
I left it as a draft to see if it might be a different way once we have the new gradle/material library included. I'll recheck it after that.

Reply:
The new gradle updates didn't help for this one. It still doesn't find the correct height without pointing it to the new R.dimen. 

I think the real issue is that the current implementation relies on the height instead of constraints (like we are doing with the ListView now) because all the parts are created in code. It might even be better to move it to a XML layout but that would be a bigger change.

As I don't like the `if` myself I think I'll close this one and add a note in the documentation how to use the workaround.